src/main/resources/configCivcraft.yml

```diff
@@ -5652,7 +5652,7 @@ recipes:
     name: Extract Aether
     type: PYLON
     production_time: 4h
-    fuel_consumption_intervall: 6m
+    fuel_consumption_intervall: 80m
```

Comment:
This allows running pylons for over a week without visiting them, which is definitely too long imo. I can see decreasing the charcoal consumption a bit, but pylon control is supposed to be tied to actual control of an area, so just building a hidden pylon in some remote location, which you only visit once a week shouldn't be possibly by design.